lib/models/measurement.js

```diff
@@ -18,9 +18,15 @@ const measurementSchema = new Schema({
     type: Date,
     required: true,
     default: moment.utc().toDate()
+  },
```

Comment:
Same here as for the box schema. You say it yourself in the comment [below](https://github.com/sensebox/openSenseMap-API/blob/ec7e558a49de496446874548be5e3db6fa5de969/lib/models/measurement.js#L28). Without testing I can imagine the design causes not only additional queries for querying the current location of the box, but for each sensor one query for the last measurement and one query for the location of the measurment.

I know we've talked about your design briefly and I suggested to store all locations non-redundant. Now I would suggest to store the location directly in the measurements and boxes.

I would be happy to discuss the topic with you further and hear your thoughts behind this design